pkg/controller/logstorage/logstorage.go

```diff
@@ -53,10 +53,11 @@ func (r *ReconcileLogStorage) createLogStorage(
 	pullSecrets []*corev1.Secret,
 	authentication *operatorv1.Authentication,
 	hdler utils.ComponentHandler,
+	hdlrPublicSecret utils.ComponentHandler,
 	reqLogger logr.Logger,
 	ctx context.Context,
 ) (reconcile.Result, bool, bool, error) {
-	var esInternalCertSecret, esCertSecret *corev1.Secret
+	var esInternalPublicCertSecret, esInternalCertSecret *corev1.Secret
```

Comment:
I don't think these name changes help, could you explain why you think these names are better?

Reply:
Their original names suggested to me that those were unrelated secrets (one is internal and the other is not), but it turns out their both `internal` in the sense that these should not to be overridden by the user.

My idea was to make these look like they are related, since one is the certificate portion of the other.

The `Public` is to make it clear this is the public certificate part of the secret (without the private key), though this secret is used by the components (maybe esgateway only) to communicate with elasticsearch. With the new naming I was trying to make it clearer in the sense of what they are, but it might be more appropriate to rename those to indicate their context.

What do you think?

Reply:
I think my suggestion from the other comment to use `KeyCert` for the secret with both and just `Cert` for the other would be better.

Reply:
Restored to original names. Related naming modifications will be moved to another change.